Makefile

```diff
@@ -12,183 +9,50 @@ DATE_FMT = +%Y%m%d%H%M.%S
 # Fallback mechanism to support other systems:
 # 1. 'date -d': Busybox and GNU coreutils.
 # 2. 'date -r': BSD date. It does not support '-d'.
-BUILD_DATE = $(shell date -u -d "@${SOURCE_DATE_EPOCH}" "${DATE_FMT}" 2>/dev/null || date -u -r "${SOURCE_DATE_EPOCH}" "${DATE_FMT}")
+export BUILD_DATE = $(shell date -u -d "@${SOURCE_DATE_EPOCH}" "${DATE_FMT}" 2>/dev/null || date -u -r "${SOURCE_DATE_EPOCH}" "${DATE_FMT}")
+
 
-GO_BUILDFLAGS ?= -trimpath -ldflags="-s -w"
+clean:
+	@rm -rf dist/
+	@docker image ls "$(DOCKER_REGISTRY)/$(DOCKER_IMAGE_NAME)*" -aq | xargs docker rmi --force
 
-ifndef GOARCH
-	GOARCH=amd64
-endif
+dist:
+	@go run github.com/goreleaser/goreleaser@v1.14.1 release --snapshot --rm-dist
```

Comment:
**suggestion**: can we skip building the docker images and move that to separate task ? the docker version requires a new dependency and it's not really used for testing/development

Reply:
https://github.com/elastic/apm-aws-lambda/pull/362/commits/2c5201016e08901ffb500210e01e7ef94b4cbc13

I re-added the `zip` rule, hope this works aswell.

Reply:
Thanks!